pkg/rbd/nodeserver.go

```diff
@@ -302,7 +302,31 @@ func getLegacyVolumeName(mountPath string) (string, error) {
 func (ns *NodeServer) mountVolumeToStagePath(ctx context.Context, req *csi.NodeStageVolumeRequest, stagingPath, devicePath string) error {
 	// Publish Path
 	fsType := req.GetVolumeCapability().GetMount().GetFsType()
-	diskMounter := &mount.SafeFormatAndMount{Interface: ns.mounter, Exec: mount.NewOsExec()}
+	source := devicePath
+
+	// Use 'ext4' as the default
+	if fsType == "" {
+		fsType = "ext4"
+	}
+	// Disk is unformatted so format it.
+	fmtArgs := []string{source}
+
+	if fsType == "ext4" {
+		fmtArgs = []string{
+			"-E nodiscard", // Do not attempt to discard blocks at mkfs time - Mainly used for RBD performance.
+			"-m0",          // Zero blocks reserved for super-user
+			"-F",           // Force flag
+			source,
```

Comment:
can we use the devicePath instead of source here, let's not create a new variable

Reply:
Done.